capabilities/version-control.md

```diff
@@ -1,28 +1,109 @@
+Comprehensive version control ensures reproducibility and traceability across the system, enabling teams to provision production-like environments, track changes, and respond quickly to evolving needs. This capability enhances disaster recovery, auditability, and quality by allowing teams to restore services efficiently, demonstrate process integrity, and receive rapid feedback on changes. When teams can scale their environments and roll back to verified working states, when necessary, they can support capacity management and defect response.
+
+Without a robust version control strategy, teams are left vulnerable to disruptions in productivity and system reliability. They may struggle to reproduce environments, leading to prolonged downtime and inefficiency. Auditability is also compromised, making regulatory compliance difficult to demonstrate. Defect response suffers, as teams lack the ability to swiftly restore a known working state. 
+
+By prioritizing effective version control practices, organizations can mitigate these risks and ensure seamless, reliable, and compliant software delivery.
+
+## Nuances
+
+This section outlines common pitfalls, challenges, or limitations teams commonly encounter when applying this capability. The goal here is not to discourage you. Rather, the goal is to arm you with the appropriate context so that you can make an informed decision about when and how to implement the capability with your teams.
+
+### Deterministic Builds vs. Up-to-Date Dependencies
+
+Version control systems often face a trade-off between deterministic builds and up-to-date dependencies. Some developers prefer specifying dependencies to use the latest versions at build time for enhanced security. However, this compromises build determinism, making it challenging to reproduce identical builds. Tools like [Dependabot](https://github.com/dependabot) can mitigate this issue by automatically creating pull requests to update dependencies, separating dependency updates from the build process.
+
+### Versioning Non-Code
+
+Version control extends beyond code to configuration, data, infrastructure, dependencies, and documentation. Establishing a version control strategy for these categories ensures a deterministic build process. This comprehensive approach enhances reproducibility, traceability, and disaster recovery. Working in a system with a deterministic build process simplifies troubleshooting, deployment, and maintenance.
+
+### Working With Large Assets
+
+Traditional version control systems like Git struggle with large file sizes. Storing large files in these systems can lead to inefficiencies, as entire file copies are stored between versions instead of diffs. This limitation necessitates specialized systems or strategies for managing large assets, ensuring efficient storage and versioning.
```

Comment:
Below we refer to Git as a "distributed version control system." I'm assuming Git is both traditional and distributed. But I'm not sure what we mean by "traditional." As opposed to...?

Reply:
Let's ditch the use of traditional altogether.

Reply:
Done.

Reply:
I'm not seeing the changes. Did you forget to push the new commit?